addons-l10n/en/remaining-replies.json

```
@@ -0,0 +1,4 @@
+{
+  "remaining-replies/reply": "reply",
+  "remaining-replies/remaining": "left"
```

Merging the two messages into
```suggestion
  "remaining-replies/reply": "reply ({replies} left)"
```
should make things simpler in the JS file and possibly for translators.